server/replicator_test.go

```diff
@@ -414,6 +431,9 @@ func TestCommitOnRestart(t *testing.T) {
 	err = client.CreateStream(ctx, subject, name, lift.ReplicationFactor(2))
 	require.NoError(t, err)
 
+	// Wait until stream is created
+	time.Sleep(5 * time.Second)
```

Comment:
We should be able to avoid a static sleep by checking state in the servers.

Reply:
fixed. Replaced with `waitForPartition`